index.html

```diff
@@ -49,15 +49,29 @@
   </head>
 
   <body>
-    <h1 id="header">Basics: Beat That! 🚀</h1>
+    <h1 id="header">🎲 Basics: Beat That! 🎲</h1>
     <div id="container">
+      <p style = "color: aliceblue;">Click the submit button to start!</p>
       <p>Input:</p>
       <input id="input-field" />
       <br />
       <button id="submit-button">Submit</button>
       <p>Output:</p>
       <div id="output-div"></div>
     </div>
+    <div id="gameinstructions">
+      <p>
+        <FONT COLOR= #9b9b9b >
+        <b style="font-size: 19px;"><center>How to play beat that❓❓</b>
+      </p>
+      <p>
+        Roll two dices and put them in order to make the highest number
+        possible.
+      </p>
+      <p>For example, if you roll a 1 and 5, your best answer would be 51.</p>
+
+      <p>This is a two-player game, so grab a friend ya scally wags! 🧞‍♀️🧞‍♂️</p></center>
+    </div>
```

Comment:
Great that you have a div for instructions here! just a small thing.

id names are generally standardised in `kebab-case` so it is best to have `game-instructions`.

also, while the `<center> center tag </center>` works, a lot of the newer browsers are starting to phase out the center tag. It's not part of the newer HTML standard.